static/index.html

```diff
@@ -8,12 +8,12 @@
     <title>Burger</title>
 </head>
 <script>
-    let stateURL = new URL("/state", window.location.origin)
-    let stopURL = new URL("/stop", window.location.origin)
+    let stateURL = new URL(window.location.pathname + "state", window.location.origin)
+    let stopURL = new URL(window.location.pathname + "stop", window.location.origin)
```

Comment:
we add `window.location.pathname` to the base of the url, because in the production verision sits behind one.sce/tv/

that way, we correctly send a query to one.sce/tv/state instead of one.sce/state